main.go

```diff
@@ -71,6 +73,8 @@ func main() {
 		URL:             coreEtcdURL,
 		CAConfigMapName: coreEtcdCAConfigMapName,
 		CertSecretName:  coreEtcdCertSecretName,
+
+		ProxyImage: proxyEtcdImage,
```

Comment:
Potential problem here. We named the struct `coreEtcdOptions` in #2, but now the struct has the `ProxyImage` field, which is not related to the core etcd at all.

What could be a better struct name?